resources/assets/js/components/Twitter.vue

```diff
@@ -74,14 +74,14 @@
         data() {
             return {
                 displayingTopTweetSince: new moment(),
-                onDisplay: [],
+                tweets: [],
                 waitingLine: [],
                 ownScreenName: '@spatie_be'
             };
         },
 
         created() {
-            this.onDisplay = JSON.parse(this.initialTweets).map(tweetProperties => new Tweet(tweetProperties));
+            this.tweets = JSON.parse(this.initialTweets).map(tweetProperties => new Tweet(tweetProperties));
```

Comment:
Conclusion: `this.initialTweets.map(...)` should work without the `JSON.parse` call!

See https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand